packaging/test_plugin_unified_storage_tiering.py

```diff
@@ -1785,3 +1785,291 @@ def test_rule_invoked_by_rodsuser_with_remote_via_irule_fails(self):
+        self.object_path = "/".join([self.collection_path, self.filename])
+
+        with session.make_session_for_existing_admin() as admin_session:
+            admin_session.assert_icommand(['iqdel', '-a'])
+
+            # Make a place for public group to put stuff.
+            admin_session.assert_icommand(["imkdir", "-p", self.collection_path])
+            admin_session.assert_icommand(["ichmod", "-r", "own", "public", self.collection_path])
+
+            session.mkgroup_and_add_users(self.group1_name, [self.user1.username, self.admin1.username])
+            # Empty group should be empty.
+            session.mkgroup_and_add_users(self.emptygroup_name, [])
+
+            lib.create_ufs_resource(admin_session, self.tier0, test.settings.HOSTNAME_2)
+            admin_session.assert_icommand(
+                ['imeta', 'add', '-R', self.tier0, 'irods::storage_tiering::group', 'example_group', '0'])
+            admin_session.assert_icommand(
+                ['imeta', 'add', '-R', self.tier0, 'irods::storage_tiering::time', str(self.tier0_time_in_seconds)])
+            admin_session.assert_icommand(
+                ['imeta', 'ls', '-R', self.tier0], 'STDOUT', [f'value: {self.tier0_time_in_seconds}', 'value: example_group'])
+
+            lib.create_ufs_resource(admin_session, self.tier1, test.settings.HOSTNAME_3)
+            admin_session.assert_icommand(
+                ['imeta', 'add', '-R', self.tier1, 'irods::storage_tiering::group', 'example_group', '1'])
+            admin_session.assert_icommand(['imeta', 'ls', '-R', self.tier1], 'STDOUT', 'value: example_group')
+
+    @classmethod
+    def tearDownClass(self):
+        if os.path.exists(self.filename):
+            os.unlink(self.filename)
+
+        self.user1.__exit__()
+        self.admin1.__exit__()
+
+        with session.make_session_for_existing_admin() as admin_session:
+            admin_session.assert_icommand(['iadmin', 'rmuser', self.user1.username])
+            admin_session.assert_icommand(['iadmin', 'rmuser', self.admin1.username])
+            admin_session.assert_icommand(['iadmin', 'rmgroup', self.group1_name])
+            admin_session.assert_icommand(['iadmin', 'rmgroup', self.emptygroup_name])
+
+            admin_session.assert_icommand(['iqdel', '-a'])
+
+            admin_session.assert_icommand(['iadmin', 'rmresc', self.tier0])
+            admin_session.assert_icommand(['iadmin', 'rmresc', self.tier1])
+            admin_session.assert_icommand(['iadmin', 'rum'])
+
+    def tearDown(self):
+        # Make sure the test object is cleaned up after each test runs.
+        with session.make_session_for_existing_admin() as admin_session:
+            admin_session.assert_icommand(["ichmod", "-M", "own", admin_session.username, self.object_path])
+            admin_session.assert_icommand(["irm", "-f", self.object_path])
+
+    def test_one_rodsadmin_with_own_succeeds(self):
```

Comment:
Consider expanding `_own_` to `_own_permission_` or `_own_perm_`.

The lack of the word, permission/perm, stuck out more when looking at the longer test names.

Reply:
Included the word "permission" in each test title.